test/parse/module/import-func.txt

```diff
@@ -8,4 +10,11 @@
   (import "math" "add" (func $add_i32 (param i32 i32) (result i32)))
   (import "test" "f32" (func $f32 (param f32) (result f32)))
   (import "test" "f64" (func $f64 (param f64) (result f64)))
-  (import "test" "i64" (func $i64 (param i64) (result i64))))
+  (import "test" "i64" (func $i64 (param i64) (result i64)))
+
+  ;; named type
+  (import "math" "add" (func $add_i32_2 (type $add_type)))
+
+  ;; named type, with repetition of type inline
+  (import "math" "add" (func $add_i32_3 (type $add_type) (param i32 i32) (result i32)))
```

Comment:
probably a different PR as this comment is about names (ps this test is good)

another to follow would be to name the parameters and check the local name section after.

```
 (import "math" "add" (func $add_i32_3 (type $add_type) (param $x i32) (param $y i32) (result i32)))
```

yet another test would be check the locals when the type reference has named params (not sure this isn't already checked)

```
(type $add_type (param $x i32) (param $y i32) (result i32)))
(import "math" "add" (func $add_i32_3 (type $add_type) (param $l i32) (param $r i32) (result i32)))
```